src/main/scala/com/databricks/spark/xml/package.scala

```diff
@@ -16,13 +16,47 @@
 package com.databricks.spark
 
 import scala.collection.Map
-
 import org.apache.hadoop.io.compress.CompressionCodec
-
```

Comment:
Actually this project follows the style guide here https://github.com/databricks/scala-style-guide#imports. Should be good to match the import order.

Reply:
Same with https://github.com/databricks/spark-xml/pull/334/files/ca711d7da1762bc88c48f1dea9fadc5b0b727c83#r244904273